recipes/highs/all/conanfile.py

```diff
@@ -34,7 +34,7 @@ def configure(self):
             self.options.rm_safe("fPIC")
 
     def layout(self):
-        cmake_layout(self, src_folder="src")
+        cmake_layout(self, src_folder="highs")
```

Comment:
thanks @galabovaa - the source directory is only used internally by the recipe, and for all recipes in Conan Center we use the `src` convention

Reply:
Thank you! I modified it back to "src"